.travis.yml

```diff
@@ -164,6 +164,9 @@ jobs:
     - stage: test
       script: ./ci/tests/dynamodb/run-tests-dynamodb.sh
       name: "DynamoDb Tests"
+    - stage: test
+      script: ./ci/tests/ehcache3/run-tests-ehcache3.sh
```

Comment:
I think it might be best to run all ehcache tests in this job, and not limit it to ehcache3 only.